drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -147,6 +147,9 @@ void at86rf2xx_tx_prepare(at86rf2xx_t *dev)
     if (state != AT86RF2XX_STATE_TX_ARET_ON) {
         dev->idle_state = state;
     }
+
+    at86rf2xx_set_state(dev, AT86RF2XX_STATE_PLL_ON);
```

Comment:
Probably also a TODO, but hard-wiring states required for OpenWSN seems like a bad idea to me.